### PR TITLE
docs: added hints to try Explore Logs (now in public preview)

### PR DESCRIPTION
**Pull request by matryer**
**What this PR does / why we need it**:

This PR adds hints to people to try Explore Logs (now in public preview) as a way of interacting with Loki.

**Checklist**
- [ ] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [ ] Tests updated
- [ ] Title matches the required conventional commits format, see [here](https://www.conventionalcommits.org/en/v1.0.0/)
  - **Note** that Promtail is considered to be feature complete, and future development for logs collection will be in [Grafana Alloy](https://github.com/grafana/alloy). As such, `feat` PRs are unlikely to be accepted unless a case can be made for the feature actually being a bug fix to existing behavior.
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/setup/upgrade/_index.md`
- [ ] For Helm chart changes bump the Helm chart version in `production/helm/loki/Chart.yaml` and update `production/helm/loki/CHANGELOG.md` and `production/helm/loki/README.md`. [Example PR](https://github.com/grafana/loki/commit/d10549e3ece02120974929894ee333d07755d213)
- [ ] If the change is deprecating or removing a configuration option, update the `deprecated-config.yaml` and `deleted-config.yaml` files respectively in the `tools/deprecated-config-checker` directory. [Example PR](https://github.com/grafana/loki/pull/10840/commits/0d4416a4b03739583349934b96f272fb4f685d15)

### CODE REVIEW

**Review by cyriltovena — APPROVED** [review]
LGTM

File: docs/sources/visualize/grafana.md [hunk 1]
```diff
@@ -3,14 +3,15 @@ title: Visualize log data
 menuTitle: Visualize
 description: Visualize your log data with Grafana
 aliases:
-   - ../getting-started/grafana/
-   - ../operations/grafana/
+  - ../getting-started/grafana/
```
Comment (matryer) [inline docs/sources/visualize/grafana.md:6]: these are just formatting

File: docs/sources/_index.md [hunk 2]
```diff
@@ -41,8 +41,12 @@ cards:
...
-Log data itself is then compressed and stored in chunks in object stores such as Amazon Simple Storage Service (S3) or Google Cloud Storage (GCS), or even locally on the filesystem.  
+Log data itself is then compressed and stored in chunks in object stores such as Amazon Simple Storage Service (S3) or Google Cloud Storage (GCS), or even locally on the filesystem.
 
 ## Explore
 
 {{< card-grid key="cards" type="simple" >}}
+
+{{< admonition type="tip" >}}
+Check out our [new queryless experience for logs](https://grafana.com/docs/grafana-cloud/visualizations/simplified-exploration/logs/) currently in Public Preview.
+{{< /admonition >}}
```
Comment (JStickler) [inline docs/sources/_index.md:52]: Landing Pages are not meant to be used for product announcements. I've already updated the Release Notes to mention Explore Logs now that it's available.

**Comment by JStickler** [comment]
@matryer I'd been meaning to review the Visualize topic, and just got my draft into a PR.  Do you mind if we close this PR in favor of #13742?

**Pull request closed without merging**